src/OpenLoco/src/GameCommands/ClearLand.cpp

```diff
@@ -18,8 +18,8 @@ namespace OpenLoco::GameCommands
     static loco_global<uint16_t, 0x00F00144> _F00144;
     static loco_global<uint8_t*, 0x00F0016C> _F0016C; // stack offset to -eventually- flags
     static loco_global<uint16_t, 0x00F00170> _F00170;
-    static loco_global<uint32_t, 0x00F003CE> _F003CE; // tile x?
-    static loco_global<uint32_t, 0x00F003D0> _F003D0; // tile y?
+    static loco_global<coord_t, 0x00F003CE> _F003CE; // tile x?
+    static loco_global<coord_t, 0x00F003D0> _F003D0; // tile y?
```

Comment:
Why not turn this into World::TilePos2, the memory layout should be compatible.

Reply:
Fwiw, #1959 gets rid of these globals entirely

Reply:
After rebasing, I've turned them into `tile_coord_t` at least, for now.